device/dell/x86_64-dellemc_z9332f_d1508-r0/DellEMC-Z9332f-M-O16C64/buffers_defaults_t0.j2

```diff
@@ -1,20 +1,57 @@
-
-{%- set default_cable = '40m' %}
-
-{%- macro generate_buffer_pool_and_profiles() %}
-    "BUFFER_POOL": {
-    },
-    "BUFFER_PROFILE": {
-    },
-{%- endmacro %}
-
-{%- macro generate_pg_profils(port_names_active) %}
-    "BUFFER_PG": {
-    },
-{%- endmacro %}
-
-{% macro generate_queue_buffers(port_names_active) %}
-    "BUFFER_QUEUE": {
-    }
-{% endmacro %}
-
+{%- set default_cable = '300m' %}
+
+{%- macro generate_port_lists(PORT_ALL) %}

```

Comment:
I see how this mapping goes with the 100G and 400G ports defined for this HW SKU.  But there are 2 10G ports as well  but I don't see that included here... is it being handled differently?